client/src/AppBundle/Resources/translations/report-contacts.en.yml

```diff
@@ -24,30 +24,30 @@ existPage:
       label: Save and continue
 
 addAnotherPage:
-  htmlTitle: "Deputy report - add contact | GOV.UK"
+  htmlTitle: "Add contact - Complete the deputy report | GOV.UK"
```

Comment:
I think this could be changed to "Add another"

Reply:
Agreed 👍🏻 changed to both the title and page heading to `Add another contact`